omc3/global_correction.py

```diff
@@ -276,6 +276,14 @@ def correction_params():
     params.add_parameter(name="update_response",
                          action="store_true",
                          help="Update the (analytical) response per iteration.", )
+    params.add_parameter(name="arc_by_arc_phase",
+                         action="store_true",
+                         help="Set to perform arc-by-arc total phase correction.", )
+    params.add_parameter(name="include_ips_in_arc_by_arc",
+                         type=str,
+                         choices=("left", "right"),
```

Comment:
Are the choices correct here since the default is `None`?

Reply:
I think "None" is always a choice when not required. I implemented this somewhen.